README.md

```diff
@@ -13,3 +13,139 @@ This is an NWB extension for storing metadata of devices used in optical experim
 
 ---
 This extension was created using [ndx-template](https://github.com/nwb-extensions/ndx-template).
+
+## Entity relationship diagram
+
+```mermaid
+%%{init: {'theme': 'base', 'themeVariables': {'primaryColor': '#ffffff', "primaryBorderColor': '#144E73', 'lineColor': '#D96F32'}}}%%
+classDiagram
+    direction BT
+    class DeviceModel{
+        <<Device>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        model : text, optional
+    }
+    class Indicator{
+        <<Device>>
+        --------------------------------------
+        attributes
+        --------------------------------------
+        label : text
+        injection_location : text, optional
+        injection_coordinates_in_mm : numeric, length 3, optional
```

Comment:
Do you have working examples of people recording the injection site in stereotactic coordinates?